bakery/src/pipelines/cops.js

```diff
@@ -244,6 +245,7 @@ const pipeline = (env) => {
       taskBakeBookMeta({ image: imageOverrides }),
       taskChecksumBook({ image: imageOverrides }),
       taskDisassembleBook({ image: imageOverrides }),
+      taskPatchDisassembledLinks({ image: imageOverrides }),
```

Comment:
Do we need to add `taskPatchDisassembledLinksSingle` to `gitDistPreviewJob` below as well?

Reply:
Yup! Forgot to add it in when I rebased!